ckanext/validation/logic/action.py

```diff
@@ -0,0 +1,406 @@
+# encoding: utf-8
+
+import logging
+import json
+
+from six import string_types
+
+import ckan.plugins.toolkit as tk
+
+from ckanext.validation.jobs import run_validation_job
+from ckanext.validation import settings
+from ckanext.validation.validation_status_helper import (
+    ValidationStatusHelper, ValidationJobAlreadyEnqueued)
+
+
+log = logging.getLogger(__name__)
+
+
+def _get_actions():
```

Comment:
This shouldn't need to start with an underscore. It's not internal to this file; it's called by the plugin module.

Reply:
\+

Reply:
This will be fixed and pushed by Alex shortly.